Sources/TuistKit/Commands/Cache/CachePrintHashesCommand.swift

```diff
@@ -16,13 +16,19 @@ struct CachePrintHashesCommand: ParsableCommand {
     )
     var path: String?
 
+    @Option(
+        name: [.customShort("P"), .long],
+        help: "The name of the cache profile."
+    )
+    var profile: String?
```

Comment:
`CachePrintHashesCommand` has the same options as `CacheWarmCommand` - rather than duplicating those, it might be useful to define [OptionGroup](https://github.com/apple/swift-argument-parser/blob/main/Examples/math/main.swift#L57) called `CacheOptions`?

Reply:
I did not know about an OptionGroup, that's great! Applied the changes.